### PR TITLE
CR-1119432 xbmgmt configure missing data printouts

### PR DESCRIPTION
**Pull request by dbenusov-xilinx**
#### Problem solved by the commit
https://jira.xilinx.com/browse/CR-1119432
This is the second half of the CR after the issue was not initially solved! Specifically the VCK5000 was not outputting data correctly. This extended to the U30 as well after further testing.

#### Bug / issue (if any) fixed, which PR introduced the bug, how it was discovered
The exception handling for the missing sysfs nodes output nothing instead of printing a not supported message. This has been present for a long while.

#### How problem was solved, alternative solutions (if any) and why they were rejected
Moved printouts and set initial values for output to better mirror the state of the devices when data is requested.

#### Risks (if any) associated the changes in the commit
None, just adding output. There is a separate issue of the security and runtime clock nodes not outputting any data if the value which should be noted in another CR.

#### What has been tested and how, request additional testing if necessary
On u280 with full support:
```
root@xsjpranjal01:~# xbmgmt configure --showx -d 65:00
0000:65:00.0
  Security level                   :
  Runtime clock scaling enabled    :
  Scaling threshold power override : 0
  Scaling threshold temp override  : 0
  Data retention                   : disabled

```
On vck5000 with no support:
```
dbenusov@xsjsagarw50:~$ xbmgmt configure --showx -d 65:00
0000:65:00.0
  Security level                   : Not supported
  Runtime clock scaling enabled    : Not supported
  Scaling threshold power override : Not supported
  Scaling threshold temp override  : Not supported
  Data retention                   : Not supported
```
On u30 with limited support:
```
dbenusov@xsjsagarw50:~$ xbmgmt configure --showx -d 17:00
0000:17:00.0
  Security level                   :
  Runtime clock scaling enabled    : Not supported
  Scaling threshold power override : Not supported
  Scaling threshold temp override  : Not supported
  Data retention                   : disabled
```
*Note the blank values! That will be a different CR that is soon to be filed.
#### Documentation impact (if any)
None

### CODE REVIEW

File: src/runtime_src/core/tools/xbmgmt2/SubCmdConfigure.cpp [hunk 1]
```diff
@@ -183,46 +183,54 @@ show_device_conf(xrt_core::device* device)
   if (is_mfg || is_recovery)
     throw xrt_core::error(std::errc::operation_canceled, "This operation is not supported with manufacturing image.\n");
 
+  boost::format sysfs_format("  %-33s: %s\n");
```
Comment (AShivangi) [inline src/runtime_src/core/tools/xbmgmt2/SubCmdConfigure.cpp:186]: sysfs is a linux specific term, can you rename it to something more generic like fmt_output

**Review by AShivangi — APPROVED** [review]
Looks good!

File: src/runtime_src/core/tools/xbmgmt2/SubCmdConfigure.cpp [hunk 2]
```diff
@@ -183,46 +183,54 @@ show_device_conf(xrt_core::device* device)
   if (is_mfg || is_recovery)
     throw xrt_core::error(std::errc::operation_canceled, "This operation is not supported with manufacturing image.\n");
 
+  boost::format node_data_format("  %-33s: %s\n");
+  std::string not_supported = "Not supported";
+  std::string sec_level = not_supported;
```
Comment (AShivangi) [inline src/runtime_src/core/tools/xbmgmt2/SubCmdConfigure.cpp:188]:
```suggestion
  uint16_t sec_level = not_supported;
```

File: src/runtime_src/core/tools/xbmgmt2/SubCmdConfigure.cpp [hunk 3]
```diff
@@ -183,46 +183,54 @@ show_device_conf(xrt_core::device* device)
   if (is_mfg || is_recovery)
     throw xrt_core::error(std::errc::operation_canceled, "This operation is not supported with manufacturing image.\n");
 
+  boost::format node_data_format("  %-33s: %s\n");
+  std::string not_supported = "Not supported";
+  uint16_t sec_level = not_supported;
```
Comment (AShivangi) [inline src/runtime_src/core/tools/xbmgmt2/SubCmdConfigure.cpp:188]:
```suggestion
  std::string sec_level = not_supported;
```

**Review by rozumx — APPROVED** [review]

**Pull request merged**